exercises/risiko/player.h

```diff
@@ -0,0 +1,91 @@
+#include<iostream>
+#include<cstdlib>
+#include<ctime>
+
+using namespace std;
+
+class risikoPlayer {
```

Comment:
a nice-to-have potrebbe essere il creare un file player.h e player.cpp, in player.h dichiari le funzioni e la classe ed implementi solo funzioni `inline`, mentre in player.cpp scrivi le implementazioni dei metodi della classe

comunque è un nice-to-have, per il resto mi sembra tutto ok, ottimo lavoro 🚀

Reply:
Provvederò a fare tutto il lavoro più tardi :)

Reply:
fallo solo se ti va di farlo 😁  👍

Reply:
Certamente, come sempre!